.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "data"]
 	path = data
 	url = git@github.com:alphagov/pay-cardid-data.git
+    branch = master
```

Comment:
Advice on submodules on the internet is to add the branch so it can track it and not get detached heads. Hence this change.